source/CMakeLists.txt

```diff
@@ -297,16 +297,14 @@ add_subdirectory(ref_base)
 add_subdirectory(angelwrap)
 add_subdirectory(cgame)
 add_subdirectory(game)
+add_subdirectory(steamshim)
```

Comment:
This is no longer conditional?
Even if there is no steam sdk?

Reply:
yes, the parent code should always be included so that builds with missing steam sdk can slot in with a precompiled steam binary and take advantage of steam features